frontend/src/views/camp/Invitation.vue

```diff
@@ -1,16 +1,29 @@
 <template>
   <auth-container>
-    <div v-if="invitationFound !== false && campCollaborationOpen !== false">
+    <div v-if="[states.INIT,states.INVITATION_FOUND].includes(dialogState)">
+      <h1 v-if="isLoggedIn" class="display-1">{{ this.$tc('components.invitation.userWelcome') }} {{ userDisplayName }}</h1>
```

Comment:
Das wäre ein perfektes Beispiel für eine Übersetzung mit Platzhalter.
```suggestion
      <h1 v-if="isLoggedIn" class="display-1">{{ this.$tc('components.invitation.userWelcome', { name: userDisplayName }) }}</h1>
```

Reply:
das ging vergessen, kommt später

Reply:
das klappt bei mir irgendwie nicht, können wir sonst mal noch schnell anschauen